src/form.ts

```diff
@@ -11,6 +11,7 @@ export class Form<T> {
   #validations: MappedValidation<T>;
   #onUpdate?: () => void;
   #field: FieldImplementation<T, T>;
+  #onChange?: (model: T) => void;
```

Comment:
```suggestion
  #onChange?: (form: Form<T>) => void;
```

this way you can get the model or just the changes or any other state about the form.